scheduler/src/cook/mesos/fenzo_utils.clj

```diff
@@ -71,7 +71,7 @@
                   (reduce summarize-placement-failure {} failures))]
     (log/debug "Job" (:job/uuid job) " placement failure summary:" summary)
     (when (and under-investigation? (seq summary))
-      [(:db/id job) summary])))
+      [(:job/uuid job) summary])))
```

Comment:
Why is this changing?

Reply:
Jobs don't have a `:db/id` after the refactoring I did a few months ago, so this will always be nil